CHANGELOG.next.asciidoc

```diff
@@ -333,6 +333,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Add input httpjson. {issue}13545[13545] {pull}13546[13546]
 - Filebeat Netflow input: Remove beta label. {pull}13858[13858]
 - Remove `event.timezone` from events that don't need it in some modules that support log formats with and without timezones. {pull}13918[13918]
+- Add YieldEventsFromField config option in the kafka input. {pull}13965[13965]
```

Comment:
Non-blocking preference: I think `ExpandEventsFromField` or even `ExpandEventListFromField` might be clearer. Either way, please document the new field in `filebeat/docs/inputs/input-kafka.asciidoc`

I like the greater flexibility of this less Azure-specific approach :-)